<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>431</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>410</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>410</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="button_sair">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>340</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>340</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>340</width>
      <height>65</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/resources/resources/logo-unp.png);
background-repeat:norepeat;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>106</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;SISTEMA DE GERENCIAMENTO DE TCC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>167</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="button_buscar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="button_alterar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Alterar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="button_cadastrar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="button_excluir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Excluir</string>
     </property>
    </widget>
    <zorder>button_buscar</zorder>
    <zorder>button_alterar</zorder>
    <zorder>button_cadastrar</zorder>
    <zorder>button_excluir</zorder>
    <zorder>button_ok</zorder>
   </widget>
   <widget class="QPushButton" name="button_ok">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>160</y>
      <width>250</width>
      <height>140</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>140</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>140</height>
     </size>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Bem-vindo ao Sistema de Gerenciamento de TCCs da Universidade Potiguar.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Esse sistema é responsável por organizar informações relativas aos Trabalhos de Conclusão de Curso dos alunos desta universidade.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Entre a opção desejada ao lado:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>600</width>
      <height>410</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>410</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>410</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resources/resources/background.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <zorder>background</zorder>
   <zorder>button_sair</zorder>
   <zorder>label</zorder>
   <zorder>button_ok</zorder>
   <zorder>label_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_titulo</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="action_sobre"/>
   </widget>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="action_sair"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuAjuda"/>
  </widget>
  <action name="action_sair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="action_sobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>button_sair</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_sair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
